### PR TITLE
Reset Workflow

### PR DESCRIPTION
**Pull request by toxic-bishop**
Reset workflow redo

### CODE REVIEW

File: lib/ui/auth/change_password_page/change_password_view.dart [hunk 1]
```diff
@@ -87,6 +88,15 @@ class ChangePasswordView extends StatelessWidget {
                                 ],
                               ),
                               SizedBox(height: 40.0.h),
+                              Text(
+                                model.errorMessage,
+                                style: TextStyle(
```
Comment (isaacadariku) [inline lib/ui/auth/change_password_page/change_password_view.dart:93]: Use the app style guide

File: lib/ui/auth/change_password_page/change_password_view.dart [hunk 2]
```diff
@@ -95,14 +105,32 @@ class ChangePasswordView extends StatelessWidget {
...
+                              SizedBox(
+                                height: 32.h,
+                              ),
+                              Padding(
+                                padding: EdgeInsets.only(left: 23.w),
+                                child: Row(
+                                  children: [
+                                    Text(
+                                      'Don\'t wish to change your password?  ',
+                                      style: TextStyle(
```
Comment (isaacadariku) [inline lib/ui/auth/change_password_page/change_password_view.dart:125]: Here too

File: lib/ui/auth/change_password_page/change_password_view.dart [hunk 3]
```diff
@@ -120,82 +148,84 @@ class ChangePasswordView extends StatelessWidget {
...
-                        color: Colors.white),
-                  ),
+              SizedBox(
+                height: 24.h,
+              ),
+              Container(
+                width: 454.w,
+                child: Text(
+                  model.isError ? model.errorSubtiltle : model.successSubtitle,
+                  style: TextStyle(
```
Comment (isaacadariku) [inline lib/ui/auth/change_password_page/change_password_view.dart:195]: Here too

File: lib/ui/auth/change_password_page/change_password_view.dart [hunk 4]
```diff
@@ -120,82 +148,84 @@ class ChangePasswordView extends StatelessWidget {
...
+              GestureDetector(
+                onTap: model.gotoLogin,
+                child: Container(
+                  height: 48.h,
+                  width: 154.w,
+                  color: Color.fromRGBO(0, 184, 124, 1),
+                  child: Center(
+                    child: Text(
+                      'Continue to Login',
+                      style: TextStyle(
```
Comment (isaacadariku) [inline lib/ui/auth/change_password_page/change_password_view.dart:215]: Here too

File: lib/ui/auth/check_email_page/check_email_view.dart [hunk 5]
```diff
@@ -39,9 +40,23 @@ class CheckEmailView extends StatelessWidget {
...
+                                ? _isResetPassword
+                                    ? model.resetTitle
+                                    : model.confirmTitle
+                                : model.resetTitle,
                             subTitle:
                                 'We sent a 6 digit code to $_email. The code expire shortly, so please enter it soon.')),
+                                verticalSpaceSmall,
+                    Text(
+                      model.errorMsg,
+                      style: TextStyle(
```
Comment (isaacadariku) [inline lib/ui/auth/check_email_page/check_email_view.dart:53]: Here too

File: lib/ui/auth/forgot_password_page/forgot_password_view.dart [hunk 6]
```diff
@@ -79,16 +83,18 @@ class ForgotPasswordView extends StatelessWidget {
...
-                            ),
+                            color: Color.fromRGBO(0, 184, 124, 1),
                             fontSize: 18.sp,
                             fontWeight: FontWeight.w600,
                             fontFamily: 'Lato'),
                       ),
                     ),
                   ),
                 ),
+                SizedBox(
```
Comment (isaacadariku) [inline lib/ui/auth/forgot_password_page/forgot_password_view.dart:94]: Use the ui_utilities available

File: lib/ui/auth/reset_password_page/reset_password_view.dart [hunk 7]
```diff
@@ -36,6 +37,15 @@ class ResetPasswordView extends StatelessWidget {
                       child: AuthHeader(
                           title: model.title, subTitle: model.subTitle)),
                   verticalSpaceLarge,
+                  Text(
+                    model.errorMessage,
+                    style: TextStyle(
```
Comment (isaacadariku) [inline lib/ui/auth/reset_password_page/reset_password_view.dart:42]: Here too

File: lib/ui/shared/dumb_widgets/goto_login/goto_login_button.dart [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+      builder: (
+        BuildContext context,
+        GotoLoginButtonModel model,
+        Widget? child,
+      ) {
+        return GestureDetector(
+          onTap: model.gotoLogin,
+          child: Text(
+            model.gotoText,
+            style: TextStyle(
```
Comment (isaacadariku) [inline lib/ui/shared/dumb_widgets/goto_login/goto_login_button.dart:22]: Here too

File: lib/ui/shared/dumb_widgets/goto_login/goto_login_button_model.dart [hunk 9]
```diff
@@ -0,0 +1,17 @@
+import 'package:stacked/stacked.dart';
+import 'package:stacked_services/stacked_services.dart';
+import 'package:zc_desktop_flutter/app/app.locator.dart';
+import 'package:zc_desktop_flutter/app/app.router.dart';
+
+class GotoLoginButtonModel extends BaseViewModel {
+  final _navigate = locator<NavigationService>();
+  final _gotoText = 'Back to Login';
```
Comment (isaacadariku) [inline lib/ui/shared/dumb_widgets/goto_login/goto_login_button_model.dart:8]: This is not needed, just put it in the UI

**Review by demolaf — DISMISSED** [review]
LGTM

File: lib/ui/shared/dumb_widgets/goto_login/goto_login_button_model.dart [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+
+  get gotoText => _gotoText;
+
+
+  void gotoLogin() {
+    _navigate.navigateTo(Routes.loginView);
+    notifyListeners();
```
Comment (isaacadariku) [inline lib/ui/shared/dumb_widgets/goto_login/goto_login_button_model.dart:15]: You don't need to call `notifyListeners` for navigation

File: lib/ui/auth/login_page/splash_view.dart [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+                Center(
+                    child: CircularProgressIndicator(
+                  color: Colors.grey,
+                ))
+              ],
+            ),
+          ),
+        );
+      },
+      onModelReady: (model) {
```
Comment (isaacadariku) [inline lib/ui/auth/login_page/splash_view.dart:36]: Use an arrow function here, since is only one function

**Review by Perpety — DISMISSED** [review]
LGTM

File: lib/services/authentication/auth_service.dart [hunk 12]
```diff
@@ -23,46 +24,83 @@ class AuthService {
...
+  Future<void> updatePassword(String password) async {
+    try {
+      await _apiService.post('/account/verify-reset-password', {"password": password, "confirm_password": password});
+    } catch (e) {
+      throw e;
+    }
+  }
 
+  void logOut() {
+    _localStorageService.removeFromDisk(_key);
```
Comment (isaacadariku) [inline lib/services/authentication/auth_service.dart:104]: What is the name of the key, you are removing here? If the name is `_key` make sure we have a readable code, by naming it properly.

**Review by isaacadariku — DISMISSED** [review]
Make the necessary changes and inform on slack

File: lib/ui/auth/login_page/splash_view.dart (hunk 11 above)
Comment (toxic-bishop) [reply lib/ui/auth/login_page/splash_view.dart]: I fixed this

File: lib/ui/shared/dumb_widgets/goto_login/goto_login_button_model.dart (hunk 10 above)
Comment (toxic-bishop) [reply lib/ui/shared/dumb_widgets/goto_login/goto_login_button_model.dart]: I fixed this

File: lib/ui/shared/dumb_widgets/goto_login/goto_login_button.dart (hunk 8 above)
Comment (toxic-bishop) [reply lib/ui/shared/dumb_widgets/goto_login/goto_login_button.dart]: I fixed this

File: lib/ui/auth/check_email_page/check_email_view.dart (hunk 5 above)
Comment (toxic-bishop) [reply lib/ui/auth/check_email_page/check_email_view.dart]: I fixed this

File: lib/ui/auth/change_password_page/change_password_view.dart (hunk 4 above)
Comment (toxic-bishop) [reply lib/ui/auth/change_password_page/change_password_view.dart]: I fixed this

File: lib/ui/auth/change_password_page/change_password_view.dart (hunk 3 above)
Comment (toxic-bishop) [reply lib/ui/auth/change_password_page/change_password_view.dart]: I fixed this

File: lib/ui/auth/change_password_page/change_password_view.dart (hunk 2 above)
Comment (toxic-bishop) [reply lib/ui/auth/change_password_page/change_password_view.dart]: I fixed this

File: lib/ui/auth/change_password_page/change_password_view.dart (hunk 1 above)
Comment (toxic-bishop) [reply lib/ui/auth/change_password_page/change_password_view.dart]: I fixed this

File: lib/ui/auth/forgot_password_page/forgot_password_view.dart (hunk 6 above)
Comment (toxic-bishop) [reply lib/ui/auth/forgot_password_page/forgot_password_view.dart]: I fixed this

File: lib/ui/auth/reset_password_page/reset_password_view.dart (hunk 7 above)
Comment (toxic-bishop) [reply lib/ui/auth/reset_password_page/reset_password_view.dart]: I fixed this

**Review by Perpety — DISMISSED** [review]
LGTM

**Review by demolaf — APPROVED** [review]
LGTM

**Review by Perpety — APPROVED** [review]
LGTM

**Pull request merged**